packages/venia-ui/lib/components/AuthBar/userChip.css

```diff
@@ -18,6 +18,11 @@
 .avatar {
 }
 
+.avatarIcon {
+    composes: root from '../Icon/icon.css';
+    --stroke: rgb(var(--venia-global-color-teal));
```

Comment:
I would avoid using teal, and go straight for the blues, so in this case:
`--venia-global-color-blue-400`